code/modules/antagonists/clockcult/clock_scriptures/scripture_judgement.dm

```diff
@@ -20,7 +20,7 @@
 	usage_tip = "The gateway is completely vulnerable to attack during its five-minute duration. It will periodically give indication of its general position to everyone on the station \
 	as well as being loud enough to be heard throughout the entire sector. Defend it with your life!"
 	tier = SCRIPTURE_APPLICATION
-	sort_priority = 8
+	sort_priority = 1
```

Comment:
Please adjust the other scripture sort_priorities of the application tier accordingly if you are changing this.
(Aka, the one that had priority 1 to 2, 2 to 3, etc, since that helps with adding / modifying stuff there)